staging/src/k8s.io/apiserver/pkg/registry/rest/rest.go

```diff
@@ -230,14 +240,14 @@ type Updater interface {
 	// Update finds a resource in the storage and updates it. Some implementations
 	// may allow updates creates the object - they should set the created boolean
 	// to true.
-	Update(ctx genericapirequest.Context, name string, objInfo UpdatedObjectInfo) (runtime.Object, bool, error)
+	Update(ctx genericapirequest.Context, name string, objInfo UpdatedObjectInfo, createValidation ValidateObjectFunc, updateValidation ValidateObjectUpdateFunc) (runtime.Object, bool, error)
```

Comment:
This breaks the patcher which is also an Updater, but obviously needs to run the mutating admission steps.

Reply:
Using a transformer should work, also for patch.